corehq/messaging/smsbackends/twilio/models.py

```diff
@@ -56,7 +56,7 @@ def get_opt_out_keywords(cls):
 
     @classmethod
     def convert_to_whatsapp(cls, number):
-        if WHATSAPP_PREFIX in number:
+        if number.startswith(WHATSAPP_PREFIX):
```

Comment:
just want to confirm there is already a check to ensure that the number would not be an integer or None?

Reply:
yep, the phone numbers in this file have gone through [clean_phone_number](https://github.com/dimagi/commcare-hq/blob/6572a4f1343b8e0d2b60dfeb299fd02ae1fdbeb3/corehq/apps/sms/util.py#L64) which returns a string.